Radzen.Blazor/RadzenDataGridColumn.razor.cs

```diff
@@ -1292,22 +1304,6 @@ internal string GetFilterOperatorSymbol(FilterOperator filterOperator)
             }
         }
 
-        /// <summary>
```

Comment:
Maybe it will be better to keep both methods and return the property value to avoid the breaking change